test/data/test_modules.py

```diff
@@ -191,73 +192,90 @@ def test_broadcast_scaled_dot_product(self):
         self.assertEqual(sdp_attn_output, sdp_attn_output_full)
         self.assertEqual(sdp_attn_weights, sdp_attn_weights_full)
 
-    def test_assert_raises(self):
-        embed_dim, nhead, tgt_len, src_len, bsz = 10, 5, 6, 10, 64
-        SDP = ScaledDotProduct()
-        query = torch.rand((tgt_len, 1, embed_dim))
-        key = value = torch.rand((src_len, 1, embed_dim))
-        attn_mask_2D = torch.randint(0, 2, (tgt_len, src_len))
+    embed_dim, nhead, tgt_len, src_len, bsz = 10, 5, 6, 10, 64
```

Comment:
Yes, I wanted to keep the variables because I think it makes the goal of the tests a lot clearer, but I agree with your suggestion, it might give the wrong pattern. I removed them.